cmd/auth/main.go

```diff
@@ -120,14 +118,6 @@ func main() {
 		}
 		tenantConfig, err := config.NewTenantConfigurationFromYAML(reader)
 		if err != nil {
-			if skyError, ok := err.(skyerr.Error); ok {
```

Comment:
This is actually needed (for ourselves). The intention here is to pretty-print the validation error to stderr so that we (the skygear team member) knows what went wrong. Can we leave a TODO here to restore this functionality when we have implemented validation error?

Reply:
I intended that the validation errors would be stored in error details, and the logger would print it out automatically. I'll add a TODO to `TenantConfiguration.Validate` to return correct validation error details.